### PR TITLE
DOC-853: Credential node overhaul E-G (except Google)

### PR DESCRIPTION
**Pull request by freakwriter**
Made the same updates here that I made in A-D. Skipped Google for now since that was more involved, but this should be everything else in these letter ranges.

### CODE REVIEW

File: docs/integrations/builtin/credentials/erpnext.md [hunk 1]
```diff
@@ -6,38 +6,46 @@ contentType: integration
...
+    - For **Cloud-hosted**:
+        - Your ERPNext **Subdomain**: Refer to the [FAQs](#how-to-find-the-subdomain-of-an-erpnext-cloud-hosted-account)
+        - Your **Domain**: Choose between `erpnext.com` and `frappe.cloud`.
+    - For **Self-hosted**:
+        - The fully qualified **Domain** where you host ERPNext
+- Choose whether to **Ignore SSL Issues**: When selected, n8n will connect even if SSL certificate validation is unavailable.
+
+If you are an ERPNext System Manager, you can also generate API keys and secrets for other users. Refer to the [ERPNext Adding Users documentation](https://docs.erpnext.com/docs/user/manual/en/adding-users){:target=_blank .external-link} for more information.
 
 ## FAQs
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/erpnext.md:44]:
```suggestion

```

File: docs/integrations/builtin/credentials/erpnext.md [hunk 2]
```diff
@@ -6,38 +6,46 @@ contentType: integration
...
 
-### How to find the subdomain of an ERPNext account?
+### How do I find the subdomain of an ERPNext cloud-hosted account?
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/erpnext.md:46]:
```suggestion
## How to find the subdomain of an ERPNext cloud-hosted account
```

File: docs/integrations/builtin/credentials/eventbrite.md [hunk 3]
```diff
@@ -6,40 +6,33 @@ contentType: integration
...
-2. Visit [your API keys page](https://www.eventbrite.com/platform/api-keys).
-3. Copy your private token.
-4. Use provided key with your Eventbrite node credentials in n8n.
+- A **Private Key**: Refer to the [Eventbrite API Authentication Get a Private Token documentation](https://www.eventbrite.com/platform/api#/introduction/authentication/1.-get-a-private-token){:target=_blank .external-link} for detailed steps to generate a Private Token. Use this private token as the **Private Key** in the n8n credential.
 
-![Getting Eventbrite credentials](/_images/integrations/builtin/credentials/eventbrite/using-access-token.gif)
+## Using OAuth2
+
+/// note | Note for n8n Cloud users
+You'll only need to enter the Credentials Name and select the **Connect my account** button in the OAuth2 credential to connect your Eventbrite account to n8n.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/eventbrite.md:35]: snippet

File: docs/integrations/builtin/credentials/eventbrite.md [hunk 4]
```diff
@@ -6,40 +6,33 @@ contentType: integration
...
+///
 
+Should you need to configure OAuth2 from scratch or need more detail on what's happening in the OAuth web flow, refer to the instructions in the [Eventbrite API authentication For App Partners documentation](https://www.eventbrite.com/platform/api#/introduction/authentication/2.-(for-app-partners)-authorize-your-users){:target=_blank .external-link} to set up OAuth.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/eventbrite.md:38]:
```suggestion
If you need to configure OAuth2 from scratch or need more detail on what's happening in the OAuth web flow, refer to the instructions in the [Eventbrite API authentication For App Partners documentation](https://www.eventbrite.com/platform/api#/introduction/authentication/2.-(for-app-partners)-authorize-your-users){:target=_blank .external-link} to set up OAuth.
```

File: docs/integrations/builtin/credentials/getresponse.md [hunk 5]
```diff
@@ -6,79 +6,54 @@ contentType: integration
...
+- A **Client ID**: Generated when you [register your application](https://apidocs.getresponse.com/v3/oauth2){:target=_blank .external-link}.
+- A **Client Secret**: Generated when you [register your application](https://apidocs.getresponse.com/v3/oauth2){:target=_blank .external-link} as the **Client Secret Key**.
+
+When you register your application, copy the **OAuth Redirect URL** from n8n and add it as the **Redirect URL** in GetResponse.
+
+/// note | Redirect URL with localhost
+The Redirect URL should be a URL in your domain, for example: `https://mytemplatemaker.example.com/gr_callback`. GetResponse doesn't accept a localhost callback URL. Refer to the [FAQs](#how-do-i-configure-oauth2-credentials-for-a-local-environment) to configure the credentials for the local environment.
+///
 
 ## FAQs
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/getresponse.md:46]:
```suggestion

```

File: docs/integrations/builtin/credentials/getresponse.md [hunk 6]
```diff
@@ -6,79 +6,54 @@ contentType: integration
...
 
-### How to configure the OAuth credentials for the local environment?
-GetResponse doesn't accept the localhost callback  URL. However, you can follow the steps mentioned below to configure the OAuth credentials for the local environment:
-1. We will use [ngrok](https://ngrok.com/) to expose the local server running on port `5678` to the internet. In your terminal, run the following command:
+### How do I configure OAuth2 credentials for a local environment?
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/getresponse.md:48]:
```suggestion
## Configure OAuth2 credentials for a local environment
```

File: docs/integrations/builtin/credentials/github.md [hunk 7]
```diff
@@ -16,59 +16,30 @@ You can use these credentials to authenticate the following nodes with GitHub.
...
+- A **User** name: Your username as it appears in your GitHub profile
+- An **Access Token**: n8n recommends using a personal access token (classic), since GitHub's fine-grained personal access tokens are still in beta and cannot access all endpoints. Refer to [Creating a personal access token (classic)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic){:target=_blank .external-link} and [Creating a fine-grained personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token){:target=_blank .external-link} for detailed steps on generating a new access token.
+
+
+## Using OAuth2
+
+/// note | Note for n8n Cloud users
+You'll only need to enter the Credentials Name and select the **Connect my account** button in the OAuth credential to connect your _Name_ account to n8n.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/github.md:42]: snippet

File: docs/integrations/builtin/credentials/gitlab.md [hunk 8]
```diff
@@ -6,46 +6,35 @@ contentType: integration
...
-5. Enter a name in the ***Name*** field.
-6. Fill out the required information to receive the access token.
-7. Select any scopes you plan to use and then click on ***Create personal access token***.
-8. Use the generated access token with your GitLab node credentials in n8n.
-9. Click on the ***Save*** button to save your credentials.
+## Using OAuth2
 
-![Getting GitLab access token](/_images/integrations/builtin/credentials/gitlab/using-access-token.gif)
+/// note | Note for n8n Cloud users
+You'll only need to enter the Credentials Name and select the **Connect my account** button in the OAuth credential to connect your _Name_ account to n8n.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/gitlab.md:37]: snippet

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
As with the previous batch: couple of small tweaks, use the same snippet for the OAuth note.

And you may have skipped FileMaker?

File: docs/integrations/builtin/credentials/gitlab.md (hunk 8 above)
Comment (freakwriter) [reply docs/integrations/builtin/credentials/gitlab.md]: Updated to snippet.

**Comment by freakwriter** [comment]
@StarfallProjects yes, I neglected to leave a note. Filemaker has a couple different APIs and I wasn't sure if we were using one specifically or both, so I opted to bump this into my "complex credentials" task and address it once I'd gotten through more of the nodes: https://linear.app/n8n/issue/DOC-842/review-more-complex-creds-pages

I'll be sure to add notes about skipped nodes in the future.

**Review by StarfallProjects — APPROVED** [review]

**Pull request merged**